libs/assets/src/lib/utils.ts

```diff
@@ -10,9 +10,8 @@ export const getAssetSymbol = (asset: AssetFieldsFragment) => {
   let symbol = asset.symbol;
 
   if (asset.source.__typename === 'ERC20') {
-    symbol = `${asset.symbol}(${
-      SUPPORTED_CHAIN_SHORT_LABELS[asset.source.chainId]
-    })`;
+    const chainLabel = getExternalChainShortLabel(asset.source.chainId);
+    symbol = `${asset.symbol}(${chainLabel})`;
```

Comment:
**nitpick (non-blocking) **: `getExternalChainShortLabel` will return `''` if the chain is unknown so this could end up being, say (`USDT()`) which would be a bit confusing. Better would be just returning the chain ID. Nitpick as it's unlikely to occur.

Reply:
Agree, and done